src/control/cmd/daos_agent/fabric.go

```diff
@@ -119,12 +144,24 @@ func (n *NUMAFabric) GetDevice(numaNode int, netDevClass uint32) (*FabricInterfa
 	n.mutex.Lock()
 	defer n.mutex.Unlock()
 
+	// If the only device is the loopback, it may not match the server's requested type, but
+	// we can try it anyway, in case the server is listening on the same machine.
+	if n.getTotalNumDevices() == 1 {
```

Comment:
What is the rationale for allowing this? If we want to be able to use loopback shouldn't we be specifying that on the server?

Reply:
I had initially taken it out for precisely that reason. We have some docker NLT cases that seem to expect it, though. Since this is destined for 2.0, I'm reluctant to make more changes than we have to.